mpv.py

```diff
@@ -1040,6 +1054,20 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def command_async(self, name, *args, callback=None):
+        """
```

Comment:
This comment has a leading newline. The first char should follow the """Immediately <-- like this.
Also, the rest of the file uses a 120-char linewidth. It would be best to keep that consistent.

Reply:
:+1: Solved in latest push.